### PR TITLE
fix: add missing negative entries in debt pool share compute

### PR DESCRIPTION
**Pull request by vbstreetz**

### CODE REVIEW

**Review by KillerByte — CHANGES_REQUESTED** [review]
we talked in private conversation and agreed since rest of queries uses `wei` library, numbers exported by this function should also use `wei`.

**Comment by vbstreetz** [comment]
> we talked in private conversation and agreed since rest of queries uses `wei` library, numbers exported by this function should also use `wei`.

awesome. on it..

**Comment by vbstreetz** [comment]
> > we talked in private conversation and agreed since rest of queries uses `wei` library, numbers exported by this function should also use `wei`.
> 
> awesome. on it..

hey! i just pushed an update to fix this.

**Comment by vbstreetz** [comment]
This has been successfully integrated in stats https://stats-git-use-synthetix-queries-pkg-synthetixio.vercel.app

Screenshot 2021-07-30 at 18 24 42

**Review by JChiaramonte7 — APPROVED** [review]
lgtm

File: packages/queries/__tests__/synths.test.ts [hunk 1]
```diff
@@ -11,6 +12,10 @@ describe('@synthetixio/queries synths', () => {
 	test('useSynthsTotalSupplyQuery', async () => {
 		const wrapper = getWrapper();
 
+		const [sETHKey, sBTCKey, sUSDKey] = [Synths.sETH, Synths.sBTC, Synths.sUSD].map(
```
Comment (KillerByte) [inline packages/queries/__tests__/synths.test.ts:15]: this constant isnt great, but its probably fine for now.

Comment (KillerByte) [reply packages/queries/__tests__/synths.test.ts:15]: instead it should let the end user application send in the list of keys they want, not hardcoded here.

File: packages/queries/package.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
 	"dependencies": {
 		"@snapshot-labs/snapshot.js": "^0.1.11",
 		"@synthetixio/contracts-interface": "^2.47.0-ovm.4",
-		"@synthetixio/data": "^2.47.0-ovm.4",
+		"@synthetixio/data": "0.0.0-3bc863c",
```
Comment (KillerByte) [inline packages/queries/package.json:37]: make sure this doesn't  get pushed

**Review by KillerByte — APPROVED** [review]
please fix a couple things but overall really appreciate :+1:

File: packages/queries/package.json (hunk 2 above)
Comment (vbstreetz) [reply packages/queries/package.json:37]: plan is to resolve this in a follow-up PR..

**Pull request merged**